nilearn/_utils/fixes/scikit_learn_validation.py

```diff
@@ -0,0 +1,400 @@
+# scikit-learn v0.17
```

Comment:
Sorry, I didnt get this. Aren't you using this file for versions lesser than 0.17 ?